docs/user-guide/deploying-your-docs.md

```diff
@@ -0,0 +1,142 @@
+3. Your `mkdocs.yml` config file and your "docs" directory (value assigned to
+the [docs_dir] configuration option) are presumed to be in the root directory of
+your project alongside your `setup.py` script.
+4. It is assumed that the default value (`"site"`) is assigned to the [site_dir]
+configuration option in your `mkdocs.yaml` config file. If you have set a
+different value, assign that value to the `--upload-dir` option.
+
+[Python]: http://www.python.org/
+[PyPI]: https://pypi.python.org/pypi
+[pythonhosted.org]: http://pythonhosted.org/
+[Setuptools]: http://pythonhosted.org/setuptools/
+[Distutils]: https://docs.python.org/2/distutils/
+[docs_dir]: configuration.md#docs_dir
+[site_dir]: configuration.md#site_dir
+
+## Other Providers
+
+Any hosting provider which can serve static files can be used to serve
+documentation generated by MkDocs. While is would be imposable to document how
+to upload the docs to every hosting provider out there, the following guidelines
+should provide some general assistance.
+
+When you build your site (using the `mkdocs build` command), all of the files
+are written to the directory assigned to the [site_dir] configuration option
+(defaults to `"site"`) in your `mkdocs.yaml` config file. Generally, you will
+simply need to copy the connects of that directory to the root directory of your
+hosting provider's server. Depending on your hosting provider's setup, you may
+need to use a graphical or command line [ftp], [ssh] or [scp] client to transfer
+the files.
+
+For example, a typical set of commands from the command line might look
+something like this:
+
+```sh
+mkdocs build --clean
+scp -r ./site usr@host:/path/to/server/root
+```
+
+Of course, you will need to replace `user` with the username you have with your
+hosting provider and `host` with the appropriate domain name. Additionally, you
+will need to adjust the `/path/to/server/root` to match the configuration of
+your hosts' file system. 
+
+[ftp]: https://en.wikipedia.org/wiki/File_Transfer_Protocol
+[ssh]: https://en.wikipedia.org/wiki/Secure_Shell
+[scp]: https://en.wikipedia.org/wiki/Secure_copy
+
+See your host's documentation for specifics. You will likely want to search
+their documentation for "ftp" or "uploading site". A list of instructions for a
+few popular hosts can be found below. MkDocs does not recommend or endorse any
+hosting providers.
+
+* [Amazon S3](http://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html)
+* [Dreamhost](http://wiki.dreamhost.com/Uploading_your_site)
+* [Blue Host](https://my.bluehost.com/cgi/help/upload-site)
+* [1&1](http://help.1and1.com/hosting-c37630/webspace-and-access-c85098/ftp-c37631/transfer-files-using-ftp-a792530.html)
+* [A Small Orange](https://kb.asmallorange.com/customer/portal/articles/1603088-upload-files-to-your-server-or-website)
+* [GoDaddy](https://support.godaddy.com/help/article/96/upload-files-to-your-website-ftp)
```

Comment:
I almost left them off for that reason - but decided to not give preferential treatment to anyone. That said, this is your project and I'm okay with whatever you want. I'll remove it.